File-Formats/Presentation/Working-with-find-and-replace.md

```diff
@@ -447,6 +520,85 @@ End Using
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PowerPoint-Examples/tree/master/Find-and-Replace/Find-and-highlight-in-PowerPoint).
 
+## Find and highlight using Regex pattern 
+
+You can find either the first occurrence or all the occurrences of a pattern of text in a PowerPoint presentation using the `Find` or `FindAll` method and highlight it.
+
+The following code example illustrates how to find all the occurrences of a pattern of text using Regex pattern and highlight it.
```

Comment:
The following code example illustrates how to find all the occurrences of a pattern of text using a Regex pattern and highlight it.